### DISCUSSION TITLE
RLS

### DISCUSSION THREAD

**Discussion by orisa** [supabase/supabase#7888] [Questions] [answered]
Im currently using the bash api for Supabase. I'm trying to enable RLS to allow only authenticated users select their inventory. I am getting an empty array with the following setup

policy name
"authenticated user can select inventory"

Target roles
"authenticated"

Using expression
auth.uid() = user_id

Expected Result: An array containing a row from the inventory table
Actual Result: An empty array

It seems I'm missing something. How can I resolve this?

**Comment by GaryAustin1** [comment, marked as answer]
For auth.uid() to work you are going to have to pass a valid jwt with the users uid in the authorization header.

**Reply by orisa** [reply]
Ok. Let me try that. Thanks
